tests/base/test_server.py

```diff
@@ -77,51 +76,57 @@ def test_version(app):
 @pytest.mark.parametrize("response_test", [
     ResponseTest(
         "http://dummy-uri/parse?q=hello",
-        [{"entities": {}, "confidence": 1.0, "intent": "greet",
-          "_text": "hello"}]
+        {u'project': None, u'entities': [], u'model': u'fallback',
```

Comment:
no `u'...'` necessary, since `from __future__ import unicode_literals` is present, all strings will be unicode automatically

Reply:
Oops. I just printed these values and pasted from that. Fixing it